.gitignore

```diff
@@ -1,13 +1,14 @@
 *.pyc
-openedx_webhooks.db
-jira.pem
-jira.pub
-jira.uuid
-docs/_build
+.cache
 .coverage
-htmlcov
 .env
-.cache
+README.md
```

Comment:
Due to my workflow, my src document is a markdown, which I don't want to accidentally check into the repo.